Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	python setup.py test
```

Comment:
If we should have a Makefile, it should be "complete" (including "install", "clean"...), but do we really need one?

Reply:
I tend to put Makefiles in every project because I often switch languages (Python, PHP, JavaScript, Perl...) and want to execute tests the same way. Running tests is the most frequently needed, followed by making docs (not included here yet). I may the Makefile later.